scipy-stubs/_typing.pyi

```diff
@@ -93,6 +93,8 @@ CorrelateMode: TypeAlias = Literal["valid", "same", "full"]
 # scipy literals
 NanPolicy: TypeAlias = Literal["raise", "propagate", "omit"]
 Alternative: TypeAlias = Literal["two-sided", "less", "greater"]
+DCTType: TypeAlias = Literal[1, 2, 3, 4]
+NormalizationMode: TypeAlias = Literal["backward", "ortho", "forward"]
```

Comment:
So you know whether these aliases will be useful outside of `scipy.fft`? 
Because if not, then it's a bit cleaner to create a dedicated `scipy.fft._typing` (and add it to [the allowlist](https://github.com/jorenham/scipy-stubs/blob/bc9eba3ad6c2d6c7be69a17dd3e6c8a54a521bf8/tests/stubtest/allowlist.txt#L1-L6))

Reply:
Oh I just realized; `scipy.fftpack` is likely to benefit from these aliases as well, so this is perfectly fine as-is 👌🏻